compiler-core/src/javascript.rs

```diff
@@ -58,10 +62,23 @@ impl<'a> Generator<'a> {
             tracker: UsageTracker::default(),
             module_scope: Default::default(),
             target_support,
+            typescript,
         }
     }
 
+    fn type_reference(&self) -> Document<'a> {
+        if self.typescript == TypeScriptDeclarations::None {
+            return Document::Str("");
+        }
+
+        let name = Document::EcoString(self.module.name.clone());
+
+        docvec!["/// <reference types=\"./", name, ".d.mts\" />", line()]
```

Comment:
What is this relative to? If it's the current file and the name of the module is for example `one/two` this would then point to `one/one/two.d.mts`, which I don't think is right.

Reply:
Should be fixed now